deploy/doc/config.md

```diff
@@ -0,0 +1,69 @@
+- [log_queries=1](https://clickhouse.tech/docs/en/operations/settings/settings/#settings-log-queries): all queries will be logged in the `system.query_log` table. Useful for debug.
+- [readonly=2](https://clickhouse.tech/docs/en/operations/settings/permissions-for-queries/#settings_readonly): do not change data on the server
+- [max_rows_to_read=200000000](https://clickhouse.tech/docs/en/operations/settings/query-complexity/#max-rows-to-read): useful if you want to prevent too broad requests
+- [cancel_http_readonly_queries_on_client_close=1](https://clickhouse.tech/docs/en/operations/settings/settings/#cancel-http-readonly-queries-on-client-close): cancel DB query when request is canceled.
+
+All these and more settings can be set in clickhouse-server configuration as user's profile settings.
+
+Useless settings:
+
+- `max_query_size`: after external data is used, the query are very short
+- `max_ast_elements`: the same
+- `max_execution_time`: with `cancel_http_readonly_queries_on_client_close=1` and `data-timeout = "1m"` it's already covered.
+
+### Index table
+See [index table](./index-table.md) documentation for details.
+
+### Index reversed queries tuning
+TODO: update after #158 or #160 is merged
+
+### ClickHouse aggregation
+For detailed description of `max-data-points` and `internal-aggregation` see [aggregation documentation](./aggregation.md).
+
+## Data tables `[[data-table]]`
+
+### Rollup
+The daemon used to use the XML-files with the same [graphite_rollup](https://clickhouse.tech/docs/en/engines/table-engines/mergetree-family/graphitemergetree/#rollup-configuration) scheme as used on ClickHouse server.
+
+For a quite long time it's recommended to use `rollup-conf = "auto"` to get the configuration from remote ClickHouse server. It will update itself each minute.
+
+If you use non `GraphiteMergeTree` family engine you can still use `rollup-conf = "auto"` and set `rollup-auto-table="graphiteMergeTreeTable"` and get the proper config. The cases when you may need it:
+
+- ReplacingMergeTree engine
+- Distributed engine
+- Materialized view
+
+It's possible as well to set `rollup-conf = "none"`. Then values from `rollup-default-precision` and `rollup-default-function` will be used.
+
+#### Rollup for reversed data tables
+When `reverse = true` is set for data-table, there are two possibles cases for [graphite_rollup](https://clickhouse.tech/docs/en/engines/table-engines/mergetree-family/graphitemergetree/#rollup-configuration):
+
+- Original regexps are used, like `^level_one.level_two.suffix$`
+- Reversed regexps are used, like `^suffix.level_two.level_one$`
+
+Depends on it to have a proper retention and aggregation you must set `rollup-use-reverted = true` for the first case and `rollup-use-reverted = false` for the second.
+
+## Carbonlink `[carbonlink]`
+The configuration to get metrics from carbon-cache. See details in [graphite-web](https://graphite.readthedocs.io/en/latest/carbon-daemons.html#carbon-relay-py) documentation.
```

Comment:
That is a bit confusing.

I would maybe say something like:

`This listener can be used by [graphite-web](...) to query data as if it would be a [carbon-cache](https://graphite.readthedocs.io/en/latest/carbon-daemons.html) daemon.`

Reply:
It's actually the client configuration for carbon-cache. I updated it a little bit. But it's clearer what is it when you see the configuration itself:

```toml
[carbonlink]
 server = ""
 threads-per-request = 10
 connect-timeout = "50ms"
 query-timeout = "50ms"
 # timeout for querying and parsing response
 total-timeout = "500ms"
```